e2e/app/definition.go

```diff
@@ -470,7 +470,7 @@ func internalEndpoints(def Definition, nodePrefix string) xchain.RPCEndpoints {
 		return endpoints
 	}
 
-	omniEVM := omniEVMByPrefix(def.Testnet, nodePrefix)
+	omniEVM := def.Testnet.BroadcastOmniEVM()
```

Comment:
lets rather change the following:
```
// omniEVMByPrefix returns a omniEVM from the testnet with the given prefix.
// Or a random omniEVM if prefix is empty.
// Or the only omniEVM if there is only one.
func omniEVMByPrefix(testnet types.Testnet, prefix string) types.OmniEVM {
```
to
```
// Or a broadcast omniEVM if prefix is empty.
```

Reply:
`internalEndpoints` is called in `Setup` and there we need to link the omniEVM to the same halo node directly, even if not available.